localstack/services/ses/ses_starter.py

```diff
@@ -0,0 +1,53 @@
+import base64
+import six
+
+from moto.ses.responses import EmailResponse as email_responses
+from moto.ses.exceptions import MessageRejectedError
+from localstack import config
+from localstack.constants import DEFAULT_PORT_SES
+from localstack.services.infra import start_moto_server
+
+
+def apply_patches():
+    def get_source_from_raw(raw_data):
+        entities = raw_data.split('\n')
+        for entity in entities:
+            if 'From: ' in entity:
```

Comment:
Wondering how robust this is. What if `raw_data` contains a string like `This is a message From: me`?

Perhaps, to make it slightly more robust:
```
re.match(r'^From: .*', entity)
```

Or even better - something like `email.message_from_string(..)` from the Python standard library could help here?

https://docs.python.org/3/library/email.parser.html#email.message_from_string